apps/web/src/views/Pools/hooks/useBoostedPoolApr.ts

```diff
@@ -1,20 +1,21 @@
-import { Address } from 'viem'
+import { getBoostedPoolApr } from '@pancakeswap/pools'
 import { useQuery } from '@tanstack/react-query'
 import { getViemClients } from 'utils/viem'
-import { getBoostedPoolApr } from '@pancakeswap/pools'
+import { Address } from 'viem'
 
 interface UseBoostedPoolApr {
   contractAddress: Address
   chainId: number | undefined
+  isFinished: boolean
```

Comment:
better called `enabled`. If the hook is not `enabled` then no need to call external endpoint

Reply:
done